WalletWasabi.Fluent/Extensions/TransactionSummaryExtensions.cs

```diff
@@ -16,7 +16,7 @@ public static bool IsConfirmed(this TransactionSummary model)
 		return confirmations > 0;
 	}
 
-	public static int GetConfirmations(this TransactionSummary model) => model.Height.Type == HeightType.Chain ? (int)Services.BitcoinStore.SmartHeaderChain.ServerTipHeight - model.Height.Value + 1 : 0;
+	public static int GetConfirmations(this TransactionSummary model) => model.Height.Type == HeightType.Chain ? (int)Services.SmartHeaderChain.ServerTipHeight - model.Height.Value + 1 : 0;
```

Comment:
This will have a conflict with your other PR https://github.com/zkSNACKs/WalletWasabi/pull/11520

Not sure how you want to proceed with this.